src/main/java/org/onedatashare/server/model/core/Resource.java

```diff
@@ -10,12 +10,19 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
+/**
+ * Resource model that holds identifiers for a file/folder of an endpoint
+ * such as file/folder path and id (for Google Drive)
+ * @param <S>
```

Comment:
Should this be more specific?

Reply:
Kept it brief since it's a parent (abstract) class. I will be adding detailed documentation for child resources in modules.

Reply:
I mean the @param